ui/yarn.lock

```diff
@@ -2302,11 +2357,25 @@
     ember-cli-babel "^7.26.6"
     ember-cli-htmlbars "^5.7.1"
 
+"@hashicorp/ember-flight-icons@^2.0.1":
```

Comment:
Looks like flight icons is in here twice

Reply:
Yeah, it's kind of confusing:
- The package is brought in already for using `FlightIcon`
- Then it's also a dependency of @hashicorp/design-system-components (use `FlightIcon` in some of the components like `Hds::IconTile`)

I ran this in terminal:

```
waypoint (hack-hds) 🌓 ~ pwd
/Users/amyrlam/src/hashicorp/waypoint/ui
waypoint (hack-hds) 🌓 ~ yarn list @hashicorp/flight-icons
yarn list v1.22.17
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ @hashicorp/design-system-components@0.1.0
│  └─ @hashicorp/flight-icons@2.1.1
└─ @hashicorp/flight-icons@2.0.0
✨  Done in 1.30s.
```

But...we've seen issues before with mismatched versions, ref: https://github.com/hashicorp/design-system-components/issues/34

For now I could bump the existing `@hashicorp/ember-flight-icons` in a separate PR, should just be adding new icons, no breaking changes for the codebase?

Reply:
Should be fine to make a PR for upgrading since we're doing it to a feature branch and not to main

Reply:
made change on https://github.com/hashicorp/waypoint/tree/amy/hack-hds-2 which will PR later